src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -20,11 +20,14 @@
 -->
 
 <template>
-	<AppContentListItem
+	<ListItem
 		:title="item.displayName"
 		:anchor-id="`conversation_${item.token}`"
-		:to="item.token ? { name: 'conversation', params: { token: item.token }} : ''"
-		:class="{ 'has-unread-messages': item.unreadMessages }"
+		:active="isActive"
+		:to="to"
+		:bold="!!item.unreadMessages"
+		:counter-number="item.unreadMessages"
+		:counter-highlighted="counterShouldBePrimary"
```

Comment:
Invalid prop: type check failed for prop "counterHighlighted". Expected Boolean, got Number with value 0.

Reply:
fixed